### ISSUE TITLE
Data::Dumper changed in core, causing errors during setup

### ISSUE THREAD

**Issue by Tux** [perl5-dbi/DBD-Oracle#136]
```
DBD-Oracle-git 501 > perl Makefile.PL
Using DBI 1.643 (for perl 5.028000 on x86_64-linux-thread-multi-ld) installed in /pro/lib/perl5/site_perl/5.28.0/x86_64-linux-thread-multi-ld/auto/DBI/
Configuring DBD::Oracle for perl 5.028000 on linux (x86_64-linux-thread-multi-ld)

If you encounter any problem, a collection of troubleshooting
guides are available under lib/DBD/Oracle/Troubleshooting.
'DBD::Oracle::Troubleshooting' is the general troubleshooting
guide, while platform-specific troubleshooting hints
live in their labelled sub-document (e.g., Win32
hints are gathered in 'lib/DBD/Oracle/Troubleshooting/Win32.pod').

Installing on a linux, Ver#4.12
Using Oracle in /usr/lib/oracle/19.13/client64
DEFINE _SQLPLUS_RELEASE = "1913000000" (CHAR)
Oracle Version 19.13.0.0 (19.13)
Found direct-link candidates: libclntsh.so
Oracle sysliblist:
Found header files in /usr/include/oracle/19.13/client64.
Your LD_LIBRARY_PATH env var is set to '/usr/lib/oracle/19.13/client64/lib:/pro/asql/o91G/bin:/pro/asql/o91G/lib:/usr/lib64:/usr/lib:/pro/po/bev/lib'

client_version=19.13


DEFINE= -Wall -Wno-comment -DUTF8_SUPPORT -DORA_OCI_VERSION=\"19.13.0.0\" -DORA_OCI_102 -DORA_OCI_112


Checking for functioning wait.ph


System: perl5.028000 linux tux 4.12.14-lp150.12.16-default #1 smp tue aug 14 17:51:27 utc 2018 (28574e6) x86_64 x86_64 x86_64 gnulinux
Compiler:   cc -O2 -D_REENTRANT -D_GNU_SOURCE -fPIC -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/pro/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_FORTIFY_SOURCE=2
Linker:     /usr/bin/ld
Sysliblist:
Linking with -lclntsh.

WARNING: Setting ABSTRACT via file 'lib/DBD/Oracle.pm' failed
 at /pro/lib/perl5/site_perl/5.28.0/ExtUtils/MakeMaker.pm line 753.
Can't dump config to mk.pm so you won't be able to build Oracle::OCI later if you wanted to: Can't use string ("Data::Dumper") as a HASH ref while "strict refs" in use at /pro/lib/perl5/site_perl/5.28.0/x86_64-linux-thread-multi-ld/Data/Dumper.pm line 664.
```
On IRC:
```
<haarg> that's a bug in DBD::Oracle that is getting exposed by the addition of strict to Data::Dumper
<haarg> it's doing Data::Dumper->Purity (1) rather than Data::Dumper->new->Purity (1)
```

**Comment by djzort** [comment]
If you'd be kind enough to make a PR i will merge it and we can do another release.

**Comment by avorop** [comment]
I've tried to fix it in my new patch. I've added PR to cand-v1.90. It contains other fixes as well, the one for ABSTRACT for example.

My problem is, I have no idea how to properly create PR, so it might be screwed up. Just in case, I've added all changes to my fork https://github.com/avorop/DBD-Oracle-1

**Comment by djzort** [comment]
I have created a v1.90 milestone and attached this issue to it

**Comment by djzort** [comment]
This should be resolved in d37ca32e80e6933fc18d9fc07f3fd4daf0fa79c6
